src/styles/row.scss

```diff
@@ -0,0 +1,10 @@
+.row {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-bottom: 46px;
+}
```

Comment:
We want to do somethinig similar to grid here, so if we have 7 cards or less they should be on the same level. Relates to checklist 13
![image](https://user-images.githubusercontent.com/71132144/225308404-a10ef7b8-1b4c-49a7-ace4-66355f7af43f.png)

Reply:
Also, the header should have border on the bottom 😄

Reply:
While I understand that, the readme states "there should always be 4 cards in a row (not 2, 3 or 5)"

Reply:
The header also does not have a border in the Figma mock-up we were given

Reply:
@SuaBy514 sorry, I meant the shadow 
<img width="752" alt="image" src="https://user-images.githubusercontent.com/71132144/225567364-cb1c3f5c-6628-4f46-823e-3876453f909f.png">

Reply:
I think the text in readme meant that it always need to be placed in the column of the same amount of cards, not different. So this I stated behaviour should be preserved

Reply:
I see, I'll see what I can do about the grid thing without actually using a grid object (since that's an assignment that's still ahead), but I expect it to take me a while